datadog/tests/cassettes/TestAccDatadogDashboardListRumEvents.yaml

```diff
@@ -0,0 +1,209 @@
+        proto_major: 1
+        proto_minor: 1
+        content_length: 698
+        transfer_encoding: []
+        trailer: {}
+        host: api.datad0g.com
+        remote_addr: ""
+        request_uri: ""
+        body: |
+            {"description":"Created using the Datadog provider in Terraform","id":"","is_read_only":true,"layout_type":"ordered","notify_list":[],"tags":[],"template_variable_presets":[],"template_variables":[],"title":"tf-TestAccDatadogDashboardListRumEvents-local-1721663521","widgets":[{"definition":{"requests":[{"columns":[{"field":"has_replay","width":"auto"},{"field":"timestamp","width":"auto"},{"field":"@view.action.count","width":"auto"}],"query":{"data_source":"rum_stream","query_string":"@type:view @session.type:user","sort":{"column":"@view.action.count","order":"desc"}},"response_format":"event_list"}],"title":"List Stream 2","title_align":"right","title_size":"16","type":"list_stream"}}]}
+        form: {}
+        headers:
+            Accept:
+                - application/json
+            Content-Type:
+                - application/json
+        url: https://api.datad0g.com/api/v1/dashboard
+        method: POST
+      response:
+        proto: HTTP/2.0
+        proto_major: 2
+        proto_minor: 0
+        transfer_encoding: []
+        trailer: {}
+        content_length: -1
+        uncompressed: true
+        body: |
+            {"id":"wwn-ema-rhd","title":"tf-TestAccDatadogDashboardListRumEvents-local-1721663521","description":"Created using the Datadog provider in Terraform","author_handle":"thibaut.gery@datadoghq.com","author_name":"Thibaut Gery","layout_type":"ordered","url":"/dashboard/wwn-ema-rhd/tf-testaccdatadogdashboardlistrumevents-local-1721663521","is_read_only":true,"template_variables":[],"widgets":[{"definition":{"requests":[{"columns":[{"field":"has_replay","width":"auto"},{"field":"timestamp","width":"auto"},{"field":"@view.action.count","width":"auto"}],"query":{"data_source":"rum_stream","query_string":"@type:view @session.type:user","sort":{"column":"@view.action.count","order":"desc"}},"response_format":"event_list"}],"title":"List Stream 2","title_align":"right","title_size":"16","type":"list_stream"},"id":705912779199021}],"notify_list":[],"created_at":"2024-07-22T15:52:03.980543+00:00","modified_at":"2024-07-22T15:52:03.980543+00:00","template_variable_presets":[],"tags":[]}
+        headers:
+            Content-Type:
+                - application/json
+        status: 200 OK
+        code: 200
+        duration: 364.835792ms
+    - id: 1
+      request:
+        proto: HTTP/1.1
+        proto_major: 1
+        proto_minor: 1
+        content_length: 0
+        transfer_encoding: []
+        trailer: {}
+        host: api.datad0g.com
+        remote_addr: ""
+        request_uri: ""
+        body: ""
+        form: {}
+        headers:
+            Accept:
+                - application/json
+        url: https://api.datad0g.com/api/v1/dashboard/wwn-ema-rhd
```

Comment:
I think you need to run this locally on production and not staging. Check the other yaml files in this repo.